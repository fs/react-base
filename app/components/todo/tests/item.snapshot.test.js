import React from 'react';
import renderer from 'react-test-renderer';
import TodoItem from '../item';

describe('TodoItem', () => {
  let props;
  const renderComponent = () => renderer.create(<TodoItem {...props} />);

  beforeEach(() => {
    props = {
      deleteTodo: () => {},
      todo: {
        id: 1,
        isComplete: false,
        name: 'Something to do',
      },
      updateTodo: () => {},
    };
  });

  it('renders correctly', () => {
    expect(renderComponent().toJSON()).toMatchSnapshot();
  });
});
