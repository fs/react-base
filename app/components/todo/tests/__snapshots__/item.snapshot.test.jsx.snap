// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoItem renders correctly 1`] = `
<button
  className="todo list-group-item"
  onClick={[Function]}
  type="button"
>
  <span>
    Something to do
  </span>
  <span
    className="glyphicon glyphicon-trash iconTrash"
    onClick={[Function]}
  />
</button>
`;
